tracer/test/Datadog.Trace.Security.IntegrationTests/IAST/AspNetCore5IastTests.cs

```diff
@@ -359,7 +359,10 @@ await VerifyHelper.VerifySpans(spansFiltered, settings)
     [SkippableTheory]
     [InlineData("/Iast/SafeCookie")]
     [InlineData("/Iast/AllVulnerabilitiesCookie")]
-    public async Task TestIastInsecureCookieRequest(string url)
+    [InlineData("/Iast/NoSameSiteCookie")]
```

Comment:
NIT: In AllVulnerabilitiesCookie, we are already testing all the vulnerabilities and also the patch. Adding individual tests for each one increments CI time and number of snapshots. Still, if you want to test them individually, I don't have a strong opinion against it.